docs/freedesktop-quick-reference.rst

```diff
@@ -101,16 +101,13 @@ Software) in order to display metadata about your application, such as a
 description, screenshots, changelogs when updates are available, and
 other miscellaneous things.
 
-Your Appdata file should be prefixed with your application's appid and
-placed in ``/app/share/metainfo/``. You should also use
-``appstream-util validate-relax`` to check your file for errors before
```

Comment:
This seems useful and shouldn't be removed?

Reply:
Yes, but it is essentially already said in the conventions docs. From this page title it is a quick reference, it can make sense to just add a link to more details (see few lines below for the link).

Reply:
there is no such thing as too much documentation. I don't think there is a benefit of removing this.

Reply:
Maybe but the risk is to have them diverging with time.
For instance in master, the file you modified in #331 (conventions.rst) now explains more things than in this file, `freedesktop-quick-reference.rst`.

A good solution is to add a link to point to this sentence in conventions (instead of copy-pasting it), as done in lines below.

Reply:
@j8r Can re-add this section so that we can merge this PR? Thanks